docs/administrator-documentation/moderne-platform/how-to-guides/agent-configuration/org-service-migration.md

```diff
@@ -1,30 +1,30 @@
 ---
-sidebar_label: Migrating out of the organizations service
-description: How to ensure a smooth migration out of the organizations service.
+sidebar_label: Organizations service migration guide
+description: How to migrate repos.csv from the org service to the agent.
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Organization Service Migration
+# Organizations service migration
 
-## What was the Organization Service?
+In order to simplify Moderne's operational complexity, we decided that configuring organizational hierarachies should only require creating a file and giving it to the Moderne agent (either via direct access or via an unauthenticated HTTP endpoint), rather than running a dedicated Organizations service.
```

Comment:
```suggestion
In order to simplify Moderne's operational complexity, we decided that configuring organizational hierachies should only require creating a file and giving it to the Moderne agent (either via direct access or via an unauthenticated HTTP/S endpoint), rather than running a dedicated Organizations service.
```